src/gl.rs

```diff
@@ -970,6 +970,20 @@ pub fn depth_range(near: f64, far: f64) {
 }
 
 #[inline]
+pub fn get_active_attrib(program: GLuint, index: GLuint) -> (i32, u32, String) {
+    let buf_size = get_program_iv(program, ffi::ACTIVE_ATTRIBUTE_MAX_LENGTH);
+    let mut name: Vec<_> = repeat(0u8).take(buf_size as usize).collect();
```

Comment:
i copied get_[program|shader]_info_log, happy to change them all but let's be consistent